src/main/java/com/dataStructures/HashTable.java

```diff
@@ -0,0 +1,45 @@
+package src.main.java.com.dataStructures;
+
+/**
+ * Hash table without repetitions
+ * @author Rick
+ * @param <E> Data type
+ * @param <T> Key type
+ */
+public class HashTable <E,T> {
+    
+    private E[] data_table;
```

Comment:
Java convention is to name variables in `camelCase`, so this should probably be `dataTable`